django/db/migrations/operations/base.py

```diff
@@ -1,6 +1,17 @@
+import enum
+
 from django.db import router
 
 
+class OperationCategory(enum.Enum):
+    ADDITION = "ADDITION"
+    REMOVAL = "REMOVAL"
+    ALTER = "ALTER"
+    PYTHON = "PYTHON"
+    SeparateDatabaseAndState = "SeparateDatabaseAndState"
+    SQL = "SQL"
```

Comment:
What do you think about adding pictogram to the enum? This should simplify `formatted_description()`
```python
    ADDITION = "+"
    REMOVAL = "-"
    ...
```
```python
    def formatted_description(self):
        description = self.describe()
        category = f"{self.category} " if self.category is not None else ""
        return f"{category}{description}"
```